packages/manager/modules/preloader/src/index.js

```diff
@@ -20,15 +21,18 @@ const buildQueryParams = (search) => {
   }, {});
 };
 
-export const attach = () => {
+export const attach = (language = 'en') => {
```

Comment:
The default displayed message will use `en` message if no user language defined or message not defined for language.